src/components/NoticiaForm.jsx

```diff
@@ -0,0 +1,103 @@
+import React, { Component } from 'react';
+import { firebaseConnect } from 'react-redux-firebase';
+import { withRouter } from 'react-router-dom';
+
+@firebaseConnect()
+@withRouter
+class NoticiaForm extends Component {
+
+  state = {
+    published: false,
+    title: '',
+    error: {
+      message: '',
+    },
+    isLoading: false,
+  }
+
+  constructor(props) {
+    super(props);
+
+    // If component recieves noticia as prop we merge it with initial state (used for editing)
+    this.state = { ...this.state, ...props.noticia };
+
+  }
+
+  componentDidMunt
+
+  addNoticia() {
+    const { title, published } = this.state;
+
+    this.setState({ isLoading: true })
+
+    this.props.firebase
+      .push('noticias', {
+        title,
+        published,
+      })
+      .then(() => {
+        this.setState({ isLoading: false })
+        this.props.history.push('/noticias');
+      })
+
+  }
+
+  updateNoticia() {
+    const { title, published } = this.state;
+
+    this.setState({ isLoading: true })
+
+    this.props.firebase
+      .update(`noticias/${this.props.id}`, {
```

Comment:
why backticks?

Reply:
its some new shiny ES6 string composition, see the variable  inside `${}`? 
[;